nautobot/core/templates/generic/object_list.html

```diff
@@ -20,14 +35,30 @@
     {% endif %}
 </div>
 <h1>{% block title %}{{ content_type.model_class|meta:"verbose_name_plural"|bettertitle }}{% endblock %}</h1>
+{% if filter_params %}
+<div class="filters-applied">
+    <b>Filters:</b>
+    {% for field, values in filter_params %}
+    <span class="select2 select2-container select2-container--bootstrap display-inline" dir="ltr">
+        <span class="selection">
+        <span class="select2-selection select2-selection--multiple p-5" role="combobox" aria-haspopup="true" aria-expanded="false" tabindex="-1" aria-disabled="false">
+            <b>{{ field }}:</b> <span class="select2-selection__clear remove-filter-param m-0" title="Remove all items" data-field-type="parent" data-field-value={{ field }}>×</span>
+            <ul class="select2-selection__rendered">
+                {% for value in values %}
+                <li class="select2-selection__choice chip" title="{{ value }}"><span class="select2-selection__choice__remove remove-filter-param" data-field-type="child"  data-field-parent={{ field }}  data-field-value={{ value }} role="presentation">×</span>{{ value }}</li>
+                {% endfor %}
+            </ul>
+        </span>
+        </span>
+        <span class="dropdown-wrapper" aria-hidden="true"></span>
+    </span> 
+    {% endfor %}
+</div>
+<hr>
+{% endif %}
+
 <div class="row">
     <div class="col-md-12">
-        {% if filter_form %}
-            <div class="col-md-3 pull-right right-side-panel noprint">
-                {% include 'inc/search_panel.html' %}
```

Comment:
From a quick grep I see that `inc/search_panel.html` is still used in:

```
nautobot/dcim/templates/dcim/connections_list.html:            {% include 'inc/search_panel.html' %}
nautobot/dcim/templates/dcim/rack_elevation_list.html:            {% include 'inc/search_panel.html' %}
nautobot/extras/templates/extras/plugins_list.html:                    {% include 'inc/search_panel.html' %}
```

Should each of those also be updated to use the new  filtering UI as well? If so, I'm fine with opening that up as a follow-up issue, just want to make sure it gets looked at.

Reply:
Yes they need to be updated